lte/gateway/deploy/roles/magma/tasks/main.yml

```diff
@@ -134,73 +135,79 @@
   when: full_provision
 
 - name: Copy Envoy config
-  copy: src=envoy.yaml dest=/var/opt/magma/envoy.yaml
+  copy:
+    src: envoy.yaml
+    dest: /var/opt/magma/envoy.yaml
   when: full_provision
 
 - name: Copy Envoy configuration script
-  copy: src=configure_envoy_namespace.sh dest=/usr/local/bin/configure_envoy_namespace.sh
+  copy:
+    src: configure_envoy_namespace.sh
+    dest: /usr/local/bin/configure_envoy_namespace.sh
   when: full_provision
 
-- name: Create symlink for sctpd binary
-  file: src='{{ c_build }}/sctpd/src/sctpd' path=/usr/local/sbin/sctpd state=link force=yes follow=no
+- name: Create symlinks for binaries
+  file:
+    src: "{{ item.src }}"
+    path: /usr/local/{{ item.path }}
+    state: link
+    force: yes
+    follow: no
+  with_items:
+    - { src: '{{ c_build }}/connection_tracker/src/connectiond', path: bin/connectiond }
+    - { src: '{{ c_build }}/dpi/dpid', path: bin/dpid }
+    - { src: '{{ c_build }}/li_agent/src/liagentd', path: bin/liagentd }
+    - { src: '{{ c_build }}/session_manager/sessiond', path: bin/sessiond }
+    - { src: '{{ c_build }}/sctpd/src/sctpd', path: sbin/sctpd }
+    - { src: '{{ go_build }}/envoy_controller', path: envoy_controller }
+    - { src: '{{ oai_build }}/oai_mme/mme', path: bin/mme }
   when: full_provision
 
 - name: Create symlink for DHCP helper CLI binary
-  file: src='/home/{{ ansible_user }}/magma/lte/gateway/python/dhcp_helper_cli/dhcp_helper_cli.py' path=/usr/local/bin/dhcp_helper_cli.py state=link force=yes follow=no
-  when: full_provision
-
-- name: Create symlink for OAI mme binary
-  file: src='{{ oai_build }}/oai_mme/mme' path=/usr/local/bin/mme state=link force=yes follow=no
-  when: full_provision
-
-- name: Create symlink for session_manager binary
-  file: src='{{ c_build }}/session_manager/sessiond' path=/usr/local/bin/sessiond state=link force=yes follow=no
-  when: full_provision
-
-- name: Create symlink for envoy controller binary
-  file: src='{{ go_build }}/envoy_controller' path=/usr/local/bin/envoy_controller state=link force=yes follow=no
-  when: full_provision
-
-- name: Create symlink for dpid binary
-  file: src='{{ c_build }}/dpi/dpid' path=/usr/local/bin/dpid state=link force=yes follow=no
-  when: full_provision
-
-- name: Create symlink for connectiond binary
-  file: src='{{ c_build }}/connection_tracker/src/connectiond' path=/usr/local/bin/connectiond state=link force=yes follow=no
-  when: full_provision
-
-- name: Create symlink for li agent binary
-  file: src='{{ c_build }}/li_agent/src/liagentd' path=/usr/local/bin/liagentd state=link force=yes follow=no
+  file:
+    src: '{{ magma_root }}/lte/gateway/python/dhcp_helper_cli/dhcp_helper_cli.py'
```

Comment:
Just to check, was it intentional that you removed `/home/{{ ansible_user }}` from the start of the path here?

Reply:
Yes. `magma_root` is defined as `/home/{{ ansible_user }}/magma` in the `magma_$NAME.yml` of the VMs.